src/structures/interfaces/InteractionResponses.js

```diff
@@ -162,6 +162,7 @@ class InteractionResponses {
    * @returns {Promise<Message|APIMessage>}
    */
   followUp(options) {
+    if(!this.deferred && !this.replied) throw new Error("INTERACTION_NOT_REPLIED");
```

Comment:
To fix ESLint warnings:
```suggestion
    if (!this.deferred && !this.replied) throw new Error('INTERACTION_NOT_REPLIED');
```